packages/engine/Specs/Scene/SkyAtmosphereSpec.js

```diff
@@ -52,9 +53,9 @@ describe(
       s.destroy();
     });
 
-    it("draws sky with setDynamicAtmosphereColor set to true", function () {
+    it("draws sky with dynamic lighting (scene light source)", function () {
       const s = new SkyAtmosphere();
-      s.setDynamicAtmosphereColor(true, false);
```

Comment:
So are SkyAtmosphere and related classes now deprecated?

Reply:
no, there are no deprecations in this PR! this was a private method called by Scene to pass globe settings to the `SkyAtmosphere`. I just changed it to take the new `DynamicAtmosphereLightingType` enum for consistency. The deprecation would happen in #11681, but not this PR.

in #11681, `SkyAtmosphere` and `Globe` may have some parameters moved to `Atmosphere`, but they still will exist for rendering the sky and terrain.